packages/components/src/components/NFT/NFT.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Box, Heading, Image, Flex, Tag, Text } from '@chakra-ui/react';
+
+export interface NFTProps {
+  tokenId: string;
+  name: string | null;
+  imageUrl: string;
+  assetContractName: string;
+  assetContractSymbol: string;
+}
+
+/**
+ * @dev Component to display an NFT given render params
+ *
+ * TODO: Add a component that fetchs the NFT data from the blockchain and uses this component to
+ * display it
+ */
+export const NFT = ({
```

Comment:
Do you think it would make sense to add a link to see it in etherscan?

Reply:
I'll iterate more on this in a future PR. i wanna look into more nft types, more metadata/attributes, link in etherscan, etc.